tasks/timesyncd.yml

```diff
@@ -4,21 +4,29 @@
   ansible.builtin.package:
     name: systemd-timesyncd
     state: present
+  when: ansible_os_family == "Debian"  
   tags:
     - systemd
     - timesyncd
     - CIS-UBUNTU2004-2.1.1.1
     - CIS-UBUNTU2004-2.1.1.2
 
+- name: Stat systemd timesyncd
+  become: true
+  ansible.builtin.stat:
+    path: /etc/systemd/timesyncd.conf
+  register: timesyncd   
+
```

Comment:
Why did this need to be added?

Reply:
I see this was added in 573b13ff3951526b98100e4d7c3eccd0379c5907 so that the modifications to the `timesyncd` configuration file are only made if the configuration file actually exists.  Can you add a comment to that effect?

Reply:
Taken care of in commit 76e0283.